core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -131,13 +129,6 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
       return UIUtils.headerSparkPage(request, stageHeader, content, parent)
     }
 
-    val storedTasks = store.taskCount(stageData.stageId, stageData.attemptId)
-    val numCompleted = stageData.numCompleteTasks
-    val totalTasksNumStr = if (totalTasks == storedTasks) {
-      s"$totalTasks"
-    } else {
-      s"$totalTasks, showing $storedTasks"
-    }
```

Comment:
@imback82 . Before removing lines, please read the commit history. For example, this is live code. Please see the following PR.
- https://github.com/apache/spark/pull/22525

Reply:
I don't think this is used, because `totalTasksNumStr` isn't. Here we'd know if it was at compile time

Reply:
The usage of `totalTasksNumStr` is removed by [this commit](https://github.com/apache/spark/commit/76ef02e499db49c0c6a37fa9dff3d731aeac9898#diff-fa4cfb2cce1b925f55f41f2dfa8c8501).

Reply:
Oh.. My bad. My memory goes stale for this part. Sorry guys.